freqtrade/tests/conftest.py

```diff
@@ -129,6 +156,21 @@ def default_conf():
                 "NEO/BTC"
             ]
         },
+        "edge": {
```

Comment:
i would not add this to the default configuration for tests, but rather extend the configuration only for tests where `edge` is actually enabled / used.

This would help catch errors / exceptions like the one i encountered while test-running ...